lib/features/hand-tool/HandTool.js

```diff
@@ -21,6 +26,31 @@ export default function HandTool(eventBus, canvas, dragging, toolManager) {
     }
   }, this);
 
+  eventBus.on('keyboard.keydown', HIGH_PRIORITY, function(e) {
```

Comment:
I have two concerns:
1. Do we actually want to implement the new shortcut in the framework? Why here and not in Camunda Modeler where the issue was created?
2. Why don't we use `Keyboard#addListener` with new `type` parameter here?

Reply:
1. I discussed this with @nikku, the answer is yes, we want this to be part of the library.
2. Yeah, the code becomes a bit more verbose but makes sense.